ci/get_global_php_version.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+php -r "echo(str_replace('-dev','',PHP_VERSION));"
```

Comment:
This needs updates. PHP_VERSION is "7.0.0RC1" now in travis. Travis has "7.1" as an alias, the nightlies are 7.2.0-dev